frontend/src/app/settings/modelsources/git-settings/edit-git-settings/edit-git-settings.component.ts

```diff
@@ -6,7 +6,7 @@
 import { Component, OnDestroy, OnInit } from '@angular/core';
```

Comment:
![image](https://user-images.githubusercontent.com/23395732/218034652-80cd3d58-5014-40ce-a66c-6f505a3af2f7.png)

Doesn't seem to work. It happens when I try to edit any instance.